storage/fs.go

```diff
@@ -55,11 +57,19 @@ func newFileSystemManager(database database) databaseFileSystemManager {
 	fm := newFlushManager(database)
 	cm := newCleanupManager(database, fm)
 
+	var jitter time.Duration
+	if maxJitter := opts.FileOpOptions().Jitter(); maxJitter > 0 {
+		nowFn := opts.ClockOptions().NowFn()
+		rand.Seed(nowFn().UnixNano())
```

Comment:
We should avoid setting the global rand seed here, we probably need to create a new rand source and then seed it.